packages/node/src/meta/ready.service.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Injectable } from '@nestjs/common';
+import { OnEvent } from '@nestjs/event-emitter';
+import { EventPayload, IndexerEvent } from '../indexer/events';
+
+@Injectable()
+export class ReadyService {
+  private ready: boolean;
+
+  constructor() {
+    this.ready = false;
+  }
+
+  @OnEvent(IndexerEvent.Ready)
+  handleReady({ value }: EventPayload<boolean>): void {
+    this.ready = value;
+  }
+
+  get getReady() {
```

Comment:
```suggestion
  get ready() {
```
getReady suggests it is a method rather a property

Reply:
this is a duplicate identifier because we already have a property in this named `ready` and thus can't have a getter with the same name. I considered using underscore notation but we don't use it elsewhere in the codebase

Reply:
underscore looks good to me